bench.sh

```diff
@@ -265,6 +265,18 @@ function tests_run_add_remove { # <test> <add?>
   fi
 }
 
+function process_external { # file
+  REGEXP="^([^ ]+) +(.*)$"
+
+  while read -r ; do
+    echo $REPLY
+    if [[ $REPLY =~ $REGEXP ]]; then
```

Comment:
It's not a "REPLY", it's a "LINE" I think.

Reply:
If I change it to $LINE, it doesn't work.

Reply:
I changed this to `read -r LINE` and updated the uses.